index.js

```diff
@@ -50,8 +50,9 @@ async function fastifyRoutesStats (fastify, opts) {
   obs.observe({ entryTypes: ['measure'], buffered: true })
 
   fastify.addHook('onRequest', function (request, reply, next) {
-    const id = request.raw.id
+    const id = request.id
     performance.mark(ONREQUEST + id)
+    reply.context.performanceMark = true;
```

Comment:
Do not modify the internal object.
`context` do not means to be used by `plugin` except `context.config`.